src/main/java/baritone/pathing/movement/MovementHelper.java

```diff
@@ -511,14 +510,14 @@ static boolean canWalkOn(BlockStateInterface bsi, int x, int y, int z) {
 
     static boolean canUseFrostWalker(CalculationContext context, BlockState state) {
         return context.frostWalker != 0
-                && state.getMaterial() == Material.WATER
+                && state.getBlock() == Blocks.WATER
                 && ((Integer) state.getValue(LiquidBlock.LEVEL)) == 0;
     }
 
     static boolean canUseFrostWalker(IPlayerContext ctx, BlockPos pos) {
         BlockState state = BlockStateInterface.get(ctx, pos);
         return EnchantmentHelper.hasFrostWalker(ctx.player())
-                && state.getMaterial() == Material.WATER
+                && state.getBlock() == Blocks.WATER
```

Comment:
You made sure mc also changed the logic like this?

Reply:
they removed materials class entirely, not quite sure if there's a better way to handle that but that does appear to be how frost walker works

Reply:
If it's how frostwalker works then it's correct.

Reply:
let me check. 1 sec

Reply:
updating
![image](https://github.com/cabaletta/baritone/assets/6234704/d504ec7e-cc5f-4e2d-b848-77460299c356)